reoptjl/models.py

```diff
@@ -2841,16 +2841,16 @@ class WIND_SIZE_CLASS_CHOICES(models.TextChoices):
         help_text="Maximum size constraint for optimization."
     )
     installed_cost_per_kw = models.FloatField(
-        default=1600,
         validators=[
             MinValueValidator(0),
             MaxValueValidator(1.0e5)
         ],
         blank=True,
-        help_text="Installed cost in $/kW"
+        null=True,
+        help_text="Installed cost in $/kW. Default determined based on size_class."
```

Comment:
The way it's set up now, Wind `installed_cost_per_kw` will get set in the Julia pkg, since it depends on the wind size class, which is also set in the Julia pkg if not input by the user. Alternatively, we could set both of these in the API code, which would allow the defaults to show up in the inputs, but would require duplicative code.